docs/confluence/advanced/grounding.md

```diff
@@ -0,0 +1,154 @@
+# Grounding 
+
+## Flag
+
+Grounding is on by default. To turn it off, use `--settings -smt_groundQuantifiers=false`
+
```

Comment:
Grounding is usually set to  “On” by default.  To turn it off, use the command `--settings -smt_groundQuantifiers=false`.